website/docs/troubleshooting/postgres/upgrade_docker.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Upgrade PostgreSQL on Docker Compose
```

Comment:
You also need to add this to the `website/sidebars.js` file, along with the other postgres upgrade item.